src/js/packages/@reactpy/client/src/vdom.tsx

```diff
@@ -68,6 +68,7 @@ function createImportSourceElement(props: {
 }): any {
   let type: any;
   if (props.model.importSource) {
+    const rootType = props.model.tagName.split(".")[0];
```

Comment:
I don't think this variable name makes much sense. Shouldn't it be something like `rootPackage`?

Reply:
I used `rootType` since this the variable two lines above is `type` - which refers to the type of Element (i.e. Component) being created. But I went ahead and changed this anyway. In fact, I decided to somewhat refactor the `createImportSourceElement` function to move a bit more logic into the `getComponentFromModule` function (renamed from `tryGetSubType`, as you requested below). Have a look at it now and see if the logic flows well.